tests/test_splicing.py

```diff
@@ -24,17 +24,17 @@ def test_splice(node_factory, bitcoind):
     result = l1.rpc.signpsbt(result['psbt'])
     result = l1.rpc.splice_signed(chan_id, result['signed_psbt'])
 
-    l2.daemon.wait_for_log(r'CHANNELD_NORMAL to CHANNELD_AWAITING_SPLICE')
-    l1.daemon.wait_for_log(r'CHANNELD_NORMAL to CHANNELD_AWAITING_SPLICE')
+    l2.daemon.wait_for_log(r'State changed from CHANNELD_NORMAL to CHANNELD_AWAITING_SPLICE')
+    l1.daemon.wait_for_log(r'State changed from CHANNELD_NORMAL to CHANNELD_AWAITING_SPLICE')
```

Comment:
```suggestion
    l2.daemon.wait_for_log(r'State changed from CHANNELD_NORMAL to CHANNELD_AWAITING_SPLICE')
    l1.daemon.wait_for_log(r'State changed from CHANNELD_NORMAL to CHANNELD_AWAITING_SPLICE')
```

Is `wait_for_log` taking a regex? So this change gets the same result as the previous one? as the postfix is the same

Reply:
It looks like I may have mixed up my branches. I added more verbose logging somewhere that adds a second log message saying something like "Moving state from X to Y", which would cause these wait_for_log's to fire too early.

But I'm switching over to checking the gossipd message explicitly, since is that is what we're really waiting for before we can route a payment.